### DISCUSSION TITLE
ترجمة مصطلح directive

### DISCUSSION THREAD

**Discussion by boussadjra** [vuejs-translations/docs-ar#138] [مصطلحات] [answered]
الـdirective هي سمات خاصة تستخدم لتوجيه طريقة إنشاء، هيكلة و سلوك العنصر/المكون:
- v-if / v-show توجه طريقة إنشاء العنصر وهيكلته.
- v-for توجه طريقة إنشاء وهيكلة العناصر.
- v-model توجه سلوك العنصر.
حسب هذا الفهم فأنا أقترح ترجمة المصطلح إلى سمة مُوجِّهة أو سمة توجيهية.
في انتظار اقتراحاتكم...

**Comment by Abdelaziz18003** [comment]
أرى أن **السمة التوجيهية** التي اقترحتها ترجمة مناسبة

**Reply by boussadjra** [reply, marked as answer]
شخصيا أميل إلى مصطلح سمة مُوجِّهة لأنه أكثر اختصارا

**Reply by Abdelaziz18003** [reply]
ليست لدي مشكلة مع هذه الترجمة أيضا، أرى أن نعتمدها بما أننا متفقين عليها
